### PR TITLE
Lynex-fusion symbol fix

### PR DESCRIPTION
**Pull request by Vasuveda**
PLEASE READ

Despite the fact that the code on DefiLlama yield-server repo master branch isn't adding the strategy between parenthesis on the pool symbol, we need this otherwise pools with the same name (but different strategy) will be shown on DefiLlama. 

Please do not close immediately if you disagree and let's have a discussion at least.

### CODE REVIEW

**Comment by llamatester** [comment]
The lynex-fusion adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       1971 passed, 1971 total
    Snapshots:   0 total
    Time:        0.685 s
    Ran all test suites.
    
    Nb of pools: 281
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────┬─────────┬────────────────┬──────────────────────────────────┬────────────────────┬──────────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────┬─────────────┐
    │ (index) │                           pool                            │  chain  │    project     │              symbol              │       tvlUsd       │       apyBase        │ apyReward │                                                                         rewardTokens                                                                          │                                        underlyingTokens                                        │  poolMeta   │
    ├─────────┼───────────────────────────────────────────────────────────┼─────────┼────────────────┼──────────────────────────────────┼────────────────────┼──────────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────┼─────────────┤
    │    0    │ '0x5db4533ecc4c455504821fa2dee56c2ea459ce37-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │        'sAMM-alUSD/USDC'         │ 58452814.47075911  │ 0.015167623210121769 │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x0E17934B9735D479B2388347fAeF0F4e58b9cc06', '0x176211869cA2b568f2A7D4EE941E073a821EE1ff' ] │ 'undefined' │
    │    1    │ '0x530071b0373ab3029cad32e0c19b75253e231b69-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │ 'weETH/WETH (Gamma Correlated)'  │ 2677098.2650569263 │  5.345261175941461   │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x1Bf74C010E6320bab11e2e5A532b5AC15e0b8aA6', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    │    2    │ '0x2babed912322c355f4b8b58ea568727892ca3f05-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDC/USDT (Steer Stable)'    │ 1715618.270114736  │  5.183614832695863   │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x176211869cA2b568f2A7D4EE941E073a821EE1ff', '0xA219439258ca9da29E9Cc4cE5596924745e12B93' ] │  'Stable'   │
    │    3    │ '0x8a9570ec97534277ade6e46d100939fbce4968f0-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'WBTC/WETH (Gamma Narrow)'    │ 1393848.4006953791 │  16.209536001695668  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x3aAB2285ddcDdaD8edf438C1bAB47e1a9D05a9b4', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    │    4    │ '0xea5abd5d15b8cd168e9eff78047db52af079e094-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │ 'wstETH/WETH (Gamma Correlated)' │ 1363777.3914986786 │  5.393466383850445   │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    │    5    │ '0x0b15a5e3ca0d4b492c3b476d0f807535f9b72079-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDC/WETH (Gamma Narrow)'    │ 1211144.4528818158 │  116.48364566729052  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x176211869cA2b568f2A7D4EE941E073a821EE1ff', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    │    6    │ '0x392ac7d2d09042ec9fafaeb8d52a88dc591e9fea-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │ 'ezETH/WETH (Gamma Correlated)'  │ 961420.5129640634  │  11.513052520844107  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x2416092f143378750bb29b79eD961ab195CcEea5', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    │    7    │ '0xad53f20917b144cd0b100dbe75988c0e4b919dba-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │  'WBTC/WETH (Yield IQ (WBTC))'   │ 741377.0581494323  │  26.49070333640614   │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x3aAB2285ddcDdaD8edf438C1bAB47e1a9D05a9b4', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'YieldIQ'  │
    │    8    │ '0x511481ef0deb10eb5c1e36b72140718c58921265-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │  'LYNX/WETH (Yield IQ (WETH))'   │ 664165.9864367812  │  109.87356826258008  │     0     │ [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60', '0x5fbdf89403270a1846f5ae7d113a989f850d1566', '0x43e8809ea748eff3204ee01f08872f063e44065f', ... 1 more item ] │ [ '0x1a51b19CE03dbE0Cb44C1528E34a7EDD7771E9Af', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'YieldIQ'  │
    │    9    │ '0xf3b1125c8505f038503e002e61a78253610d4f60-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDT/WETH (Gamma Narrow)'    │ 564281.6602855948  │  119.7212615542445   │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0xA219439258ca9da29E9Cc4cE5596924745e12B93', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │  'Narrow'   │
    └─────────┴───────────────────────────────────────────────────────────┴─────────┴────────────────┴──────────────────────────────────┴────────────────────┴──────────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────┴─────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by slasher125** [comment]
pls move that additional context to poolMeta

**Comment by Vasuveda** [comment]
@slasher125 I'm sorry, what do you mean?

**Comment by slasher125** [comment]
this `(${found.title})` needs to go into `poolMeta`

**Comment by Apegurus** [comment]
> this `(${found.title})` needs to go into `poolMeta`

@slasher125 

I believe you may be misunderstanding how our pools work. We integrate with multiple liquidity managers, these means that using just token0/token1 for `symbol` produces duplicated undifferentiated symbols. Currently looking at the yields page, you can see that many of our strategies are not there. Which I would assume it is because it just does not show the same symbol twice. 

This change is to differentiate the strategies by ALM at the symbol level. 

Regards.

**Comment by slasher125** [comment]
the only unique value is `pool`, you can have many duplicates on symbol. however, the symbol field is required to only contain deposit token symbols. any further context, such as fee tiers, strategies etc goes into `poolMeta`. we will display poolMeta as defined in the adapter on the ui. clear?

File: src/adaptors/lynex-fusion/index.js [hunk 1]
```diff
@@ -60,7 +59,7 @@ const getApy = async () => {
     );
     const poolMeta = `${found?.type}`
     const symbol = found
```
Comment (Vasuveda) [inline src/adaptors/lynex-fusion/index.js:61]: .

Comment (Vasuveda) [reply src/adaptors/lynex-fusion/index.js:61]: Why is there a discrepancy between code on defillama's yield-server repo and UI? Pool "sAMM-alUSD-USDC" is shown with a "(undefined)" on the UI but according to code that shouldn't happen. It's fixed apparently with the PR as seen on the test suites results.
Or are the parenthesis displaying the poolMeta?

Comment (slasher125) [reply src/adaptors/lynex-fusion/index.js:61]: > Or are the parenthesis displaying the poolMeta?

yes

Comment (slasher125) [reply src/adaptors/lynex-fusion/index.js:61]: @Vasuveda all clear now?

Comment (Vasuveda) [reply src/adaptors/lynex-fusion/index.js:61]: Yes! Thank you for clarification. Will fix it soon then.

**Comment by llamatester** [comment]
The lynex-fusion adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       1992 passed, 1992 total
    Snapshots:   0 total
    Time:        0.685 s
    Ran all test suites.
    
    Nb of pools: 284
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────┬─────────┬────────────────┬───────────────────┬────────────────────┬─────────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┬────────────────────────────────────────────────────────────────────────────────────────────────┬────────────────────┐
    │ (index) │                           pool                            │  chain  │    project     │      symbol       │       tvlUsd       │       apyBase       │ apyReward │                                                                         rewardTokens                                                                          │                                        underlyingTokens                                        │      poolMeta      │
    ├─────────┼───────────────────────────────────────────────────────────┼─────────┼────────────────┼───────────────────┼────────────────────┼─────────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────┼────────────────────┤
    │    0    │ '0x5db4533ecc4c455504821fa2dee56c2ea459ce37-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │ 'sAMM-alUSD/USDC' │ 73734229.16354479  │ 0.03889776692180037 │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x0E17934B9735D479B2388347fAeF0F4e58b9cc06', '0x176211869cA2b568f2A7D4EE941E073a821EE1ff' ] │     'Classic'      │
    │    1    │ '0x530071b0373ab3029cad32e0c19b75253e231b69-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │   'weETH/WETH'    │ 2339144.5312863705 │ 0.9919937417421846  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x1Bf74C010E6320bab11e2e5A532b5AC15e0b8aA6', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │ 'Gamma Correlated' │
    │    2    │ '0x2babed912322c355f4b8b58ea568727892ca3f05-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDC/USDT'    │  1637106.89825683  │  5.291415955637207  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x176211869cA2b568f2A7D4EE941E073a821EE1ff', '0xA219439258ca9da29E9Cc4cE5596924745e12B93' ] │   'Steer Stable'   │
    │    3    │ '0x8a9570ec97534277ade6e46d100939fbce4968f0-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'WBTC/WETH'    │ 1339807.5403523375 │  26.32085297881065  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x3aAB2285ddcDdaD8edf438C1bAB47e1a9D05a9b4', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │   'Gamma Narrow'   │
    │    4    │ '0x0b15a5e3ca0d4b492c3b476d0f807535f9b72079-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDC/WETH'    │ 1118094.7340663315 │  114.4306519285461  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x176211869cA2b568f2A7D4EE941E073a821EE1ff', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │   'Gamma Narrow'   │
    │    5    │ '0x392ac7d2d09042ec9fafaeb8d52a88dc591e9fea-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │   'ezETH/WETH'    │ 893760.9159995648  │ 6.2063423022967354  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x2416092f143378750bb29b79eD961ab195CcEea5', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │ 'Gamma Correlated' │
    │    6    │ '0xea5abd5d15b8cd168e9eff78047db52af079e094-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │   'wstETH/WETH'   │ 819535.6509137207  │ 2.0986109766543772  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │ 'Gamma Correlated' │
    │    7    │ '0x511481ef0deb10eb5c1e36b72140718c58921265-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'LYNX/WETH'    │ 557980.9760089486  │  78.4699380818801   │     0     │ [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60', '0x5fbdf89403270a1846f5ae7d113a989f850d1566', '0x43e8809ea748eff3204ee01f08872f063e44065f', ... 1 more item ] │ [ '0x1a51b19CE03dbE0Cb44C1528E34a7EDD7771E9Af', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │ 'Yield IQ (WETH)'  │
    │    8    │ '0xf3b1125c8505f038503e002e61a78253610d4f60-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │    'USDT/WETH'    │ 530669.3357070371  │ 115.98776681760562  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0xA219439258ca9da29E9Cc4cE5596924745e12B93', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │   'Gamma Narrow'   │
    │    9    │ '0xcffbfd665bedb19b47837461a5abf4388c560d35-lynex-fusion' │ 'Linea' │ 'lynex-fusion' │   'STONE/WETH'    │ 459808.4407561284  │ 11.250612653777756  │     0     │                                                       [ '0x63349ba5e1f71252ecd56e8f950d1a518b400b60' ]                                                        │ [ '0x93F4d0ab6a8B4271f4a28Db399b5E30612D21116', '0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f' ] │ 'Gamma Correlated' │
    └─────────┴───────────────────────────────────────────────────────────┴─────────┴────────────────┴───────────────────┴────────────────────┴─────────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┴────────────────────────────────────────────────────────────────────────────────────────────────┴────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by Vasuveda** [comment]
@slasher125 Fix pushed. Hope this time we get it right

**Pull request merged**